application/views/collection_instrument_view.py

```diff
@@ -31,10 +31,7 @@ def before_collection_instrument_view():
 def upload_collection_instrument(exercise_id, ru_ref=None):
     file = request.files["file"]
     classifiers = request.args.get("classifiers")
-    if current_app.config["SEFT_GCS_ENABLED"] is False:
-        instrument = CollectionInstrument().upload_instrument(exercise_id, file, ru_ref=ru_ref, classifiers=classifiers)
-    else:
-        instrument = CollectionInstrument().upload_to_bucket(exercise_id, file, ru_ref=ru_ref, classifiers=classifiers)
+    instrument = CollectionInstrument().upload_to_bucket(exercise_id, file, ru_ref=ru_ref, classifiers=classifiers)
```

Comment:
Not for this PR, as you haven't changed it, but because the highlight made me see it, but the instantiation/creation and the upload should be separate, the method upload_to_bucket should not really be returning an instrument. The naming would suggest an action returning bool

Reply:
I can make a card to improve it, as I think you have a point, but this PR is just reorganising code and removing a flag.  I don't want to increase the scope of it unless there was a really good reason for it.